### DISCUSSION TITLE
hard to find Copilot subscription info

### DISCUSSION THREAD

**Discussion by tomasohara** [community/community#49102] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

It is very difficult to find the Copilot subscription info. Please add a modern search facility to Github as this is a problem in general (i.e., navigating the convoluted menu system).

p.s., Why so many loops to add feedback? This is forcing it into a formal issue

**Comment by iamcjwilliams** [comment]
@tomasohara Thank you for your feedback 🚀
